explain how to avoid errors when overriding translations (#1698)
plone/documentation: docs/i18n-l10n/translating-text-strings.md

zcml = my.package
```

> When doing so, you may need to add the following zcml stanza in your package's `configure.zcml` file:

Reviewer: ```suggestion
When doing so, you may need to add the following ZCML stanza in your package's {file}`configure.zcml` file:
```

```xml

> 

Reviewer: ```suggestion
```

<include package="Products.CMFPlone" />

> 

Reviewer: ```suggestion
```

```

> This **must** go *after* the `registerTranslations` stanza, before any other registration you may have in your package.

Reviewer: ```suggestion
This **must** go *after* the `registerTranslations` stanza, and before any other registration you might have in your package.
```

> It should look like this:

Reviewer: ```suggestion
The registration should look like the following example.
```
